redisinsight/ui/src/pages/rdi/pipeline/pages/jobs/Jobs.spec.tsx

```diff
@@ -82,4 +83,29 @@ describe('Jobs', () => {
 
     expect(screen.getByTestId('rdi-pipeline-transformation-link')).toHaveAttribute('href', 'https://docs.redis.com/latest/rdi/data-transformation/data-transformation-pipeline/')
   })
+
+  it('should send telemetry event with proper data', () => {
```

Comment:
There is also a `TelemetryPageView.RDI_JOBS` event that does seem to have a unit test. Doesn't look like the event was added with this PR but writing a test for it should be quick and easy.

Reply:
Probably you miss it, please, take a look at 29 line
`  it('should call proper sendPageViewTelemetry', () => {`

Reply:
I did miss it. My mistake.